CONTRIBUTING.md

```diff
@@ -20,7 +20,11 @@ Ever since Moulberry has stopped working on NEU, other contributors have been wo
 
 ### Software configuration
 
-- Clone the NEU repository using `git clone https://github.com/NotEnoughUpdates/NotEnoughUpdates`.
+- Fork the NEU repository using the fork button on top right of the page and name the repo NotEnoughUpdates.
+- Clone the forked repository using `git clone https://github.com/<YourUserName>/NotEnoughUpdates`.
+- Make sure to create new branches for features you are working and not commit to the master branch of your repository
```

Comment:
actually, please be consistent with the dots at the end of sentences

Reply:
Ok Let me do that